test/distributed/test_store.py

```diff
@@ -275,12 +275,26 @@ def _create_store_with_ws(self, addr, world_size):
             addr, world_size, wait_for_workers=False, use_libuv=self._use_libuv
         )
 
-    def test_address_already_in_use(self):
-        err_msg_reg = "^The server socket has failed to listen on any local "
+    def test_invalid_port(self):
+        addr = DEFAULT_HOSTNAME
+        port = 1
+
+        err_msg_reg = f"^(The server socket has failed to listen on any local|The server socket has failed to bind).*{port}"
```

Comment:
Why are we providing two options for assertion checks? Should it just be `...failed to bind`?

Reply:
The non-libuv backend just says "failed to listen" where as libuv backend is more specific and prints failed to bind.

Reply:
make sense!